[oneTBB] Fix flow graph inaccuracies (#367)
oneapi-src/oneAPI-spec: source/elements/oneTBB/source/flow_graph/graph_cls.rst

    Resets the graph according to the specified flags.
    Flags to ``reset()`` can be combined with bitwise-``or``.

>     Note: ``reset()`` is thread-unsafe operation, don't use it in concurrent environment

Reviewer: ```suggestion
    .. note::
        ``reset()`` is a thread-unsafe operation, don't call it concurrently.
```
